pkg/api/customization/cluster/formatter.go

```diff
@@ -145,6 +148,12 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 		}
 	}
 
+	nodes, err := f.nodeLister.List(resource.ID, labels.Everything())
```

Comment:
resource.ID will the clusterID, does it also act as the namespaceID for the List call?

Reply:
Yes, the v3.nodes live in the cluster namespace